tests/cephfs/ceph-83574161.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+
+    """
+    Test operation:
+    1. Create a random subvolume_group name that does not exist
+    2. Try to create a subvolume on the subvolume_group name
+    3. If the creation fails, memo it
+    4. Try to delete a subvolume on the subvolume_group name
+    5. If the deletion fails, memo it.
+    6. If the creation and deletion are failed, return 0
+    """
+    try:
+
+        tc = "CEPH-83574161"
+        log.info(f"Running CephFS tests for BZ-{tc}")
+        fs_util = FsUtils(ceph_cluster)
+        clients = ceph_cluster.get_ceph_objects("client")
+        client1 = clients[0]
+        fs_details = fs_util.get_fs_info(client1)
+        result = {}
+        if not fs_details:
+            fs_util.create_fs(client1, "cephfs")
+        fs_util.auth_list([client1])
+        subvolume_group = "non_exist_subvolume_group_name"
+        subvolume_name_generate = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(5))
+        )
+        subvolume = {
+            "vol_name": "cephfs",
+            "subvol_name": f"subvol_{subvolume_name_generate}",
+            "size": "5368706371",
+            "group_name": f"{subvolume_group}",
+        }
+
+        result["result"] = 0
```

Comment:
I think we can just use "result" variable instead of "result["result"]"

Reply:
@yogesh-mane I have changed it to some number to gahter results of the each tests